instrumentation/github.com/labstack/echo/otelecho/test/echo_test.go

```diff
@@ -202,3 +200,59 @@ func TestErrorNotSwallowedByMiddleware(t *testing.T) {
 	err := h(c)
 	assert.Equal(t, assert.AnError, err)
 }
+
+func TestSpanNameFormatter(t *testing.T) {
+	imsb := tracetest.NewInMemoryExporter()
+	provider := trace.NewTracerProvider(trace.WithSyncer(imsb))
+
+	tests := []struct {
+		name     string
+		method   string
+		path     string
+		url      string
+		expected string
+	}{
+		// Test for standard methods
+		{"", http.MethodGet, "/user/:id", "/user/123", "GET /user/:id"},
```

Comment:
Just noticed. That you have forgotten to name the test cases 😉

Reply:
I still want to be a bit lazy; I'll add this part later. 😂